mp/game/momentum/resource/ui/SettingsPanel_HudSettings.res

```diff
@@ -121,10 +151,40 @@
         "auto_tall_tocontents" "1"
         "use_proportional_insets"		"0"
         "Default"		"0"
-        "cvar_name"		"mom_hud_speedometer_hvel"
-        "cvar_value"		"0"
+        "cvar_name"		"mom_hud_speedometer_showenterspeed"
+        "cvar_value"		"1"
         "Font"		"DefaultVerySmall"
     }
+	"SpeedoShowUnitLabels"
+	{
+        "ControlName"		"CvarToggleCheckButton"
+        "fieldName"		"SpeedoShowUnitLabels"
+        "xpos"		"0"
+        "ypos"		"0"
+        "autoResize"		"0"
+        "pinCorner"		"0"
+        "pin_to_sibling" "SpeedoShowStageEnter"
+        "pin_corner_to_sibling"		"0"
+        "pin_to_sibling_corner"		"2"
+        "visible"		"1"
+        "enabled"		"1"
+        "tabPosition"		"0"
+        "labelText"		"#MOM_Settings_Speedometer_Unit_Labels"
+        "textAlignment"		"west"
+        "dulltext"		"0"
+        "brighttext"		"0"
+        "wrap"		"0"
+        "centerwrap"		"0"
+        "textinsetx"		"6"
+        "textinsety"		"0"
+        "auto_wide_tocontents"		"1"
+        "auto_tall_tocontents" "1"
+        "use_proportional_insets"		"0"
+        "Default"		"0"
+        "cvar_name"		"mom_hud_speedometer_unit_labels"
+        "cvar_value"		"1"
```

Comment:
Since this is a new thing you're adding I would remove the `cvar_value` from just this section, and worry about the rest of the ones in your #639 PR.

Reply:
Removed all the cvars in this file using find/replace.. woops. Shouldn't effect #639 very much though.